doc/classes/Sprite2D.xml

```diff
@@ -44,7 +44,7 @@
 			<return type="bool" />
 			<param index="0" name="pos" type="Vector2" />
 			<description>
-				Returns [code]true[/code], if the pixel at the given position is opaque and [code]false[/code] in other case.
+				Returns [code]true[/code], if the pixel at the given position in local coordinates is opaque and [code]false[/code] in other case.
```

Comment:
```suggestion
				Returns [code]true[/code], if the pixel at the given position, in local coordinates, is opaque and [code]false[/code] in other case.
```
Or
```suggestion
				Returns [code]true[/code], if the pixel at the given position is opaque and [code]false[/code] in other case. The position is in local coordinates.
```
I'd argue the assumption generally is that things are in local space for generality, but since this is already specified for `get_rect` it makes sense to add here too

Reply:
The grammar is wrong.

in other case -> otherwise

Remove the comma after "true"

Reply:
They don't have to change that. But I _will_ softly slap whoever wrote the original descriptions for these.